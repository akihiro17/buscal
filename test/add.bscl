var i1: i64 = 1;
var i2: i64 = 1;
var i3: i64 = 0;
i3 = i1 + i2 + 10;
echo(i3);

var s1: str = "this ";
var s2: str = "is";
var s3: str = " a pen";
s3 = s1 + s2 + s3 + ".";
echo(s3);
