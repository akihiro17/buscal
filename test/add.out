12
this is a pen.
